apps/server/src/modules/management/controller/api-test/database-management.api.spec.ts

```diff
@@ -79,15 +79,17 @@ describe('Database Management Controller (API)', () => {
 			expect(result.status).toEqual(201);
 		});
 		it('should encrypt plain text', async () => {
-			const txt = faker.string.alphanumeric(42);
+			const plainText = faker.string.alphanumeric(42);
+			const key = faker.string.alphanumeric(16);
+			const body = { plainText, key };
+
 			const result = await request(app.getHttpServer())
 				.post(`/management/database/encrypt-plain-text`)
-				.set('content-type', 'text/plain')
-				.set('content-length', Buffer.byteLength(txt).toString())
-				.send(txt);
+				.set('content-type', 'application/json')
+				.send(body);
 
 			expect(result.status).toEqual(200);
-			expect(result.text).not.toHaveLength(0);
+			expect(result.text).toHaveLength(100);
```

Comment:
maybe add a simple test comparing the resulted string to a given encrypted string?
This would ensure the test is valid for this encryption only, so if the algorithm changes again, it might still have length 100 but completely different result.

Given the hard-coded string and key, expect the result to equal 'xxx''.